ui-tests/src/test/java/org/openmrs/reference/SearchActiveVisitsTest.java

```diff
@@ -31,7 +30,7 @@ public void setup() {
         new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
     }
 
-    @Ignore
+ 
```

Comment:
@sherrif10 did you just forget to add @Test annotation to this test.

Reply:
Thanks @kdaud Resolved it